ui/perfherder/compare/CompareTableControls.jsx

```diff
@@ -207,6 +250,10 @@ CompareTableControls.propTypes = {
     PropTypes.bool,
   ]),
   onPermalinkClick: PropTypes.func,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
I'm curious how this linting rule being hit. Is there a bug with the rule? This doesn't seem like a rule that should need to be disabled.

Reply:
It may be a bug with the rule. Locally it does not detect that I use it. I'm going to delete the comments for disabling the rules and see if it breaks further in the CI

Reply:
Unfortunately there is an error in the CI if I don't have the rule disabled there. 
![lint-error](https://user-images.githubusercontent.com/51707553/72452926-06c7d300-37c7-11ea-8c30-3b357791bf5b.png)

Reply:
@sarah-clements I disabled the rule back

Reply:
Where is `retriggerJob` being used? I only see `retriggerJobs` is being used in this file. Could I persuade you to rename `retriggerJobs` to something like `retriggerMultipleJobs` to make this easier to distinguish? 

If you are just passing through `retriggerJob` from `CompareViewControls` via {...this.props} you don't need to declare that prop here unless its used. So if that's the case you can list the `retriggerJob` propType in the component that will be using it.

Reply:
I renamed `retriggerJobs` to `retriggerMultipleJobs` and also removed the props that required us to disable the lint rule